docs/reference/controls/itemsrepeater.md

```diff
@@ -15,7 +15,7 @@ The items repeater is a port of the UWP `ItemsRepeater` control. For further inf
 :::
 
 :::info
-The items repeater was moved [here](https://www.nuget.org/packages/Avalonia.Controls.ItemsRepeater)
+The `ItemsRepeater` was moved inside an additional _Avalonia UI_ package. To use the `ItemsRepeater` in your project, you must reference the [Avalonia.Controls.ItemsRepeater](https://www.nuget.org/packages/Avalonia.Controls.ItemsRepeater) _NuGet_ package. Check this [PR](https://github.com/AvaloniaUI/Avalonia/pull/14989) for additional information.
```

Comment:
Might want to change terminology. "additional" -> "external".

Also, the PR is not the correct one. That's a recent PR that moves `ItemsRepeater` into a separate repo as the long-term plan is probably to deprecate the component (v12 timeframe) in favor of `ItemsControl` improvements.

It moved into a separate package in v11 previews with this one: https://github.com/AvaloniaUI/Avalonia/pull/10112

Reply:
> Might want to change terminology. "additional" -> "external".

What's wrong with additional? They say the same in the DataGrid info badge [here](https://docs.avaloniaui.net/docs/reference/controls/datagrid/).

> Also, the PR is not the correct one

Thanks, fixed in 149f5f1.

Reply:
IMO, "external" is better for clarity as it isn't a core package. I have a lot of opinionated gripes on the terminology present in docs. Not enough to hold up a change though. LGTM.